docs/concepts/pdp/overview.mdx

```diff
@@ -0,0 +1,248 @@
+---
+sidebar_position: 1
+title: Overview
+---
+
+# Permit's PDP (Policy Decision Point)
+
+A PDP is a network node responsible for answering authorization queries using policies and contextual data.
+Hence, PDPs should be highly available, performant and physically close to the querying services to minimize latency and ensure high availability.
+
+Permit's PDP essentially becomes your microservice for authorization, bundling together OPA, OPAL and an API Server.
+
+The PDP is easy to install via Docker image, and will ensure **zero-latency, great performance, high availability, and improved security**.
+Like all of Permit's customer deployed components, the [Permit PDP is open-source](https://github.com/permitio/PDP) and is available publicly from [Docker hub](https://hub.docker.com/r/permitio/pdp-v2).
+
+## Deploying the PDP
+
+Permit.io supports all PDP layouts and provides the missing layers on-top of open-source PDP solutions (such as OpenPolicyAgent). These layers include policy delivery and updating, supporting data collection, application level SDKs, application level instrumentation and more.
+Read about [the various layouts you can deploy the PDP with here](/how-to/deploy/overview#pdp-deployments).
+
+There are several ways you can deploy the PDP:
+
+
+### Managed Cloud PDP
+
+Permit.io provides a managed Cloud PDP at `https://cloudpdp.api.permit.io` which is available globally.
```

Comment:
you need to add important note that this "cloud pdp" only supports RBAC ( doesn't support ReBAC / ABAC )